src/components/Greetings.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+function Greetings(props) {
+    const {lang, text} = props
+    if(lang === 'de') {
```

Comment:
Good job here, Miguel! Another way to do that would be using the switch statement.

```
    switch(lang) {
        case 'en':
            return <h1>Hello {children}</h1>
        case 'es':
            return <h1>Hola {children}</h1>
        case 'fr':
            return <h1>Bonjour {children}</h1>
        case 'de':
            return <h1>Hallo {children}</h1>
        default:
            return <h1>{children}</h1>
    }
```